calico-enterprise/threat/web-application-firewall.mdx

```diff
@@ -244,58 +256,118 @@ By default WAF will not block a request even if it has matching rule violations.
 | On     | Denies HTTP traffic. {{prodname}}  will log the event in Security Events. | Yes          |
 | Off    | Be cautious about using this option. Traffic is not denied, and there are no events. |No                                                                      | Yes         |
 
-#### Change anomaly scoring threshold
 
-The default scoring threshold is set to 100, a high value that is unlikely to result in blocked requests, even if blocking mode is enabled. A critical rule violation scores 5 by default so a single HTTP request would need many rule violations to pass the default threshold of 100. You are encouraged to fine-tune and lower this threshold value iteratively until you are happy with the requests being allowed and denied for your cluster.
+##### Other basic customizations
 
-1. Edit the `crs-setup.conf` file and set `tx.inbound_anomaly_score_threshold` to the value of your choice:
-   ```bash
-     SecAction \
-      "id:900110,\
-      phase:1,\
-      nolog,\
-      pass,\
-      t:none,\
-      setvar:tx.inbound_anomaly_score_threshold=100,\
-      setvar:tx.outbound_anomaly_score_threshold=100"
-   ```
+For basic customizations, it's best to add it after all the includes in `tigera.conf`. In fact, this is the reason why the `SecRuleEngine` directive and the rest of [our customizations](https://github.com/tigera/operator/blob/master/pkg/render/applicationlayer/embed/coreruleset/tigera.conf#L8-L17) are situated there.
 
-These are the default scoring points for each severity level, applied to each rule individually.
-The lower the overall anomaly scoring threshold value is, the more likely it is that traffic will be denied.
+An example is adding a sampling mode. For that, the tigera.conf will look like this:
 
-| Severity | Score                                                                                                                                                             | Description |
-| ------ | ----------------------------------------------------------------------------------------------------------------------------------------------------------------------- | ----------- |
-| Critical | 5 | Mostly generated by the application attack rules (93x and 94x files)
-| Error     | 4 | Generated mostly from outbound leakage rules (95x files) |
-| Warning    | 3 | Generated mostly by malicious client rules (91x files) |
-| Notice    | 2 | Generated mostly by the protocol rules (92x files) |
+```conf
+# CoreRuleset activation
```

Comment:
```suggestion
# Core Rule Set activation
```
Make this change only if this is your comment, and not a comment that appears in the actual code. If that form is baked in, we should leave it as it actually appears.